frontend/.eslintrc.js

```diff
@@ -1,17 +1,203 @@
 'use strict';
 
-const { configs } = require('@nullvoxpopuli/eslint-configs');
-const { baseConfig: baseNode } = require('@nullvoxpopuli/eslint-configs/configs/node');
+function hasDep(depName) {
+  try {
+    return Boolean(require.resolve(depName));
+  } catch (e) {
+    if (e.message.startsWith(`Cannot find module '${depName}'`)) return false;
 
-const config = configs.ember();
+    throw e;
+  }
+}
+
+function proposedEmberDefault(personalPreferences) {
+  let hasTypeScript = hasDep('typescript');
+  let noPrettier = process.env.NO_PRETTIER;
+
+  const configBuilder = {
+    modules: {
+      browser: {
+        get js() {
```

Comment:
every "leaf object" in this structure defines an ESLint "overrides" entry.

Docs here: https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-based-on-glob-patterns